project/dimensions/subsectors.csv

```diff
@@ -0,0 +1,38 @@
+id,name
+unspecified,Unspecified
+bev_compact,BEV Compact
+bev_midsize,BEV Midsize
+bev_pickup,BEV Pickup
+bev_suv,BEV SUV
+phev_compact,PHEV Compact
+phev_midsize,PHEV Midsize
+phev_pickup,PHEV Pickup
+phev_suv,PHEV SUV
+bev_light_medium,BEV Light-Medium
+bev_medium,BEV Medium
+bev_non_freight_truck,BEV Non-freight truck
+bev_heavy,BEV Heavy
+bev_school_bus,BEV School Bus
+bev_transit_bus,BEV Transit Bus
+light_rail,Light Rail
```

Comment:
Transportation, please revise. I guesstimated these from the current TEMPO dataset and our DECARB Task 6 approach document.

Reply:
These look good!